src/mca/ptl/base/ptl_base_frame.c

```diff
@@ -67,8 +69,18 @@ pmix_ptl_API_t pmix_ptl = {
     .stop_listening = pmix_ptl_base_stop_listening
 };
 
+static int max_msg_size = PMIX_MAX_MSG_SIZE;
```

Comment:
We should probably change this from an `int` to a `size_t` since that is what we compare against later.
I don't expect us to ever have a user ask for a `max_msg_size` of greater than 2GB.

Reply:
oh - sure! will do right now

Reply:
it's going to be the same in the v2.1 branch, so i'll push a change there too